README.md

```diff
@@ -11,7 +11,7 @@
 
 **StaticArrays** provides a framework for implementing statically sized arrays
 in Julia (≥ 0.5), using the abstract type `StaticArray{Size,T,N} <: AbstractArray{T,N}`.
-Subtypes of `StaticArray` will provide fast implementations of common array and
+Subtypes of [StaticArray](@ref) will provide fast implementations of common array and
```

Comment:
What is this `@ref` for?

Reply:
It inserts a link to the documentation of the `StaticArray` type.

Reply:
Right, but is the README.md actually used during the Documenter build process?  I would have thought the README shouldn't necessarily go in the full docs, and it doesn't appear to be mentioned in docs/make.jl.  (I need to run Documenter for myself, just haven't got around to it yet)

Reply:
Plain documenter ignores README.md, and this link is documenter markup, not markup.

Reply:
Ah I didn't see that this was the README file. Thanks for pointing out!